code-challenges.js

```diff
@@ -1,63 +1,131 @@
-// ASSESSMENT 2: Coding Practical Questions
+//  ASSESSMENT 2: Coding Practical Questions
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-// Use the test variables provided.
+//  Use the test variables provided.
 
 var num1 = 15
-// Expected output: "15 is divisible by three"
+//  Expected output: "15 is divisible by three"
 
 var num2 = 0
 // Expected output: "0 is divisible by three"
 
 var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
+//  Expected output: "-7 is not divisble by three"
+
+//create a function that takes in an argument
+const div3 = (number) => {
+   //create an if statement that uses the modulo operator to see if a number is divisible by three
+    if( number % 3 === 0 ){
+        return `${number} is divisible by three`
+        }else{
+            return `${number} is not divisible by three`
+        }
+    }
+    //console log the function 
+     console.log(div3(num1))
+     console.log(div3(num2))
+     console.log(div3(num3))
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
-
-
-
+//  Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
+    
+//First you will declare a function that takes in an argument
+    const nounCap = (array) => {
+        //create a local variable that maps through the array
+        let eachNoun = array.map(value =>{
+            //capitalize the first index of each word
+            //then connect it to the rest of the word with substring method
+            return value[0].toUpperCase() + value.substring(1)
+            })
+            return eachNoun.join(" ")
+            
+         
+        
+    }
```

Comment:
great! this could be refactored a bit more but i really like how clear your process was here. good job!